README.md

```diff
@@ -31,10 +31,13 @@ This project tests all builds against each supported version of Fedora, the late
 
 ## Out of scope
 
-* Signing and pushing images to various image storages.
-  See [Skopeo](https://github.com/containers/skopeo/).
-* Container Runtimes daemons for working with the Kubernetes CRI interface.
+* Container runtimes daemons for working with the Kubernetes CRI interface.
   See [CRI-O](https://github.com/kubernetes-sigs/cri-o).
+* Supporting `docker-compose`.  We believe that Kubernetes is the defacto
+  standard for container orchestration.  Hence, Podman allows to create PODs
+  from a Kubernetes YAML file. See
+  [podman-play-kube](https://github.com/containers/libpod/blob/master/docs/podman-play-kube.1.md)
```

Comment:
Link generate too

Reply:
Are you going to add generate too per @mheon's suggestion?

Reply:
Sorry, the PR is in already. Shall I update? I think that the `podman play` manpage could mention it as well.

Reply:
Sure open a new PR.